src/pages/components/FavoriteGrid.qml

```diff
@@ -51,17 +75,13 @@ SilicaGridView {
         }
     }
 
-    readonly property bool contextMenuActive: contextMenu && contextMenu.active
-    property Item contextMenu: currentItem ? currentItem._menuItem : null
-    // Figure out which delegates need to be moved down to make room
-    // for the context menu when it's open.
-    property int minOffsetIndex: contextMenu ? currentIndex - (currentIndex % columns) + columns : 0
-    property int yOffset: contextMenu ? contextMenu.height : 0
-
     delegate: ListItem {
         id: container
 
         property Item remorse
+        property real offsetY: largeScreen
+                               ? (((y % (favoriteGrid.pageHeight-container.contentHeight+1)) / (favoriteGrid.pageHeight-container.contentHeight+1)) - 0.5) * (Theme.paddingLarge*4)
```

Comment:
This will cause the actual grid item (ListItem) and the visible item (FavoriteItem) y coordinate to diverge, and eventually  the visible item will "disappear" before it reaches the bottom of the screen.